### DISCUSSION TITLE
[Feature] Ability for search to search against partner shared files

### DISCUSSION THREAD

**Discussion by adoreparler** [immich-app/immich#2703] [Feature Request]
### The feature

I would like the ability to search across partner sharing when using the search or facial recognition. Either by having it as a option or it just does it without an option.
 
* When I search for a term it searches my library along with any shared libraries.
* When I click on a facial recognition picture it shows photos from both my library along with any shared libraries.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by argus-core** [comment]
This feature would be very useful for the approach I'm currently testing:
- 2x personal users
- A separate dummy user with an imported, read-only library shared with both personal users

**Comment by BenjaminDo** [comment]
Hello,

Do you know why this feature request has been closed ? 
Is it already implemented or on the work ? 

Thanks !
